### DISCUSSION TITLE
Can I retarget and compile to .NET 7?

### DISCUSSION THREAD

**Discussion by michaellashinsky** [EvanMulawski/FanControl.Razer#5] [Q&A] [answered]
I know, I know I can try to clone and compile this with a new target myself and see if it works, but its been a while since I have played in .NET land.  

Do you see anything that would prevent me from doing this or any gotchas?

**Comment by EvanMulawski** [comment, marked as answer]
Hi @michaellashinsky, I am planning to multi-target the plugin in the next few weeks (and automate the build and release process).

The only gotcha is the [Synchronization](https://github.com/EvanMulawski/FanControl.Razer/tree/main/src/EMRazer.Synchronization) library: .NET 7 and .NET Framework differ significantly here regarding mutex access control. (The mutex is not technically required for this plugin, but I implemented it to provide interoperability for others and it's the same implementation used in my CorsairLink plugin.)

**Reply by michaellashinsky** [reply]
Thanks for the heads up @EvanMulawski
